tests/ServerRequestTest.php

```diff
@@ -313,6 +312,10 @@ public function dataGetUriFromGlobals()
                 'https://www.blakesimpson.co.uk/blog/article.php?id=10&user=foo',
                 array_merge($server, ['HTTPS' => 'on', 'SERVER_PORT' => '443']),
             ],
+            'No HTTPS param' => [
```

Comment:
I've added a test called "With HTTPS param = 1" to make sure we still test the old "Normal request".